classes/Product.php

```diff
@@ -164,7 +164,9 @@ class ProductCore extends ObjectModel
     /** @var string|array Meta description or array of meta description by id_lang */
     public $meta_description;
 
-    /** @var string|array Meta keywords or array of meta keywords by id_lang */
+    /**
+     * @deprecated
+     */
     public $meta_keywords;
```

Comment:
@matks should have something to say about this

Reply:
@jolelievre When building this Command, Julius found out that **in current product page (177) there is no way to manage meta tags**. So the fields are in still in ObjectModel but they cannot be reached from BO.

We explored this and found out that meta tags are now 100% useless for SEO so we believe
- they were removed from BO on purpose
- they were left in ObjectModel, undeprecated, either for backward compatibility or because it was forgotten

This is why we agreed to "finish the job" by deprecating them. But this PR does not remove any feature, it only marks the field as deprecated.